### PR TITLE
Task-36530: Make Global site and pages accessible by /platform/externals group members

### PR DESCRIPTION
**Pull request by azayati**

### CODE REVIEW

File: extension/war/src/main/webapp/WEB-INF/conf/social-extension/portal/portal/global/portal.xml [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <portal-config>
     <portal-name>global</portal-name>
     <locale>en</locale>
-    <access-permissions>*:/platform/users</access-permissions>
+    <access-permissions>*:/platform/users;*:/platform/externals</access-permissions>
```
Comment (plamarque) [inline extension/war/src/main/webapp/WEB-INF/conf/social-extension/portal/portal/global/portal.xml:5]: isn't it dangerous to provide such wide access to external users ? The global navigaiton contains pages that they are not authorized to see (such as people, spaces and others). How is the permission check is done ? at page level ?

Comment (azayati) [reply extension/war/src/main/webapp/WEB-INF/conf/social-extension/portal/portal/global/portal.xml:5]: You have anticipated, despite this configuration, right now all global pages are accessible to only /platform/users. The goal is not to give externals full access to global site pages, the work is not yet finished and the appropriate permissions will be given effectively at page level.

Comment (plamarque) [reply extension/war/src/main/webapp/WEB-INF/conf/social-extension/portal/portal/global/portal.xml:5]: > the work is not yet finished and the appropriate permissions will be given effectively at page level.

OK. Just make sure you don't push this change before you safeguard page permissions.

**Review by ahmadmido — APPROVED** [review]
LGTM

**Pull request merged**